src/PriceIndicator/PriceIndicator.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+import { PriceIndicator } from './PriceIndicator';
+import { Tooltip } from '../Tooltip';
+
+export default {
+  title: 'Components/PriceIndicator',
+  argTypes: { onChange: { action: 'changed' } },
+  component: PriceIndicator,
+  parameters: {
+    backgrounds: {
+      default: 'orange',
+    },
+  },
+} as Meta;
+
+export const Default: Story = () => {
+  const [priceTotal, setPriceTotal] = useState(0);
+  const [priceIncrease, setPriceIncrease] = useState(0);
+
+  const updatePrice = ({ increase }: { increase: number }): void => {
+    setPriceIncrease(increase);
+    setPriceTotal(prevState => prevState + increase);
+  };
+  return (
+    <div style={{ height: '100px' }}>
+      <button type="button" onClick={() => updatePrice({ increase: Math.random() })}>
+        Increase price by random penny`&apos;`s
```

Comment:
can it just be penny's? look weird in the output

Reply:
ESlint complains and says it should be escaped with either this code or another couple of similar suggestions

Reply:
Think it just needs to be "&apos;" rather than "`&apos;`" without the double quotes.